### ISSUE TITLE
Feature Request: compare configs with output

### ISSUE THREAD

**Issue by djpanda** [FezVrasta/ark-server-tools#697]
I was thinking of an Switch for printconfig, maybe "-c" who compares alls the configs like arkmanager.cfg instance.cfg and the gameusersettings.ini in the gamefolder and return double entries and wrong entries with missing flags in the arkmanager own configfiles like "ark_" or "arkflag_" dont know if it would be possible but i ran into it that i have doubles in my configs because of i had to learn how exactly this tool is working and what files are the right ones to set the configs and because of try and error it seems there are left back a few doubles and also wrong entries. I can do it manualy but a nice way would be to have a Switch for printconfig who does my work :)

**Comment by klightspeed** [comment]
`ark_` options are specified in the `...?opt1=val?opt2=val?...` command-line string.  Many options to ShooterGameServer can be specified either in that string or as `-opt1=val`.
Note that ShooterGameServer writes many of the options specified on the commandline to GameUserSettings.ini, and thus duplicate detection there would be useless.
The settings in the instance config override the settings in arkmanager.cfg, so you could provide common options between instances into arkmanager.cfg, and override them per instance.

**Comment by djpanda** [comment]
yes i knowed that but it was only ment for clean looking configs without doubles and wrong entries not for functional.. i know that if there are doubles the gameusersettings will win and instance will always overwrite the global arkmanager.cfg

**Comment by klightspeed** [comment]
Options specified on the commandline (via arkmanager.cfg or instance.cfg) will override and overwrite settings in GameUserSettings.ini

**Comment by Uskaanax** [comment]
Just use diff. https://linux.die.net/man/1/diff
